spec/acceptance/basic_spec.rb

```diff
@@ -1,17 +1,28 @@
-require 'beaker-puppet'
-require 'beaker-pe'
 require 'spec_helper_acceptance'
 
 describe 'basic tests:' do
   it 'make sure we have copied the module across' do
     # No point diagnosing any more if the module wasn't copied properly
-    shell('ls /etc/puppetlabs/code/modules/puppetdb') do |r|
+    run_shell('ls /etc/puppetlabs/code/environments/production/modules/puppetdb') do |r|
```

Comment:
I doubt that this check brings any value :D

Reply:
i have a whole set of acceptance changes for the next PR. This was part of #344. My next changes remove this test.